### PR TITLE
Redux+Zustandt guides: add s to synchronize and make it explicit that Live structures cannot be used

### PR DESCRIPTION
**Pull request by FlowFlorent**

### CODE REVIEW

File: docs/pages/get-started/react-redux.mdx [hunk 1]
```diff
@@ -278,7 +278,7 @@ can be nested to create the state that you want.
   servers is the winner.
 
 When using our redux integration you don't need to interact directly with these
```
Comment (FlowFlorent) [inline docs/pages/get-started/react-redux.mdx:280]: should this be more clear? "don't need" can leave room for interpretation that you can use the Live data structures.

File: docs/pages/get-started/react-zustand.mdx [hunk 2]
```diff
@@ -268,7 +268,7 @@ can be nested to create the state that you want.
   servers is the winner.
 
 When using our Zustand integration you don't need to interact directly with
```
Comment (FlowFlorent) [inline docs/pages/get-started/react-zustand.mdx:270]: same as https://github.com/liveblocks/liveblocks/pull/704/files#r1113223634

File: docs/pages/get-started/react-redux.mdx (hunk 1 above)
Comment (nvie) [reply docs/pages/get-started/react-redux.mdx:280]: Yeah, you could add that you _cannot_ interact directly with these datastructures. Good call!

**Review by nvie — APPROVED** [review]
Great addition of `s`'es. A+, would review again.

File: docs/pages/get-started/react-redux.mdx (hunk 1 above)
Comment (FlowFlorent) [reply docs/pages/get-started/react-redux.mdx]: Thank you! Adjusted

File: docs/pages/get-started/react-redux.mdx [hunk 3]
```diff
@@ -277,9 +277,9 @@ can be nested to create the state that you want.
   property simultaneously, the last modification received by the Liveblocks
   servers is the winner.
 
-When using our redux integration you don't need to interact directly with these
-data structures. Our enhancer synchronize your store with our data structures
-based on the `storageMapping` configuration.
+When using our redux integration you cannot to interact directly with these data
```
Comment (nvie) [inline docs/pages/get-started/react-redux.mdx:280]:
```suggestion
When using our redux integration you cannot interact directly with these data
```

**Review by nvie — APPROVED** [review]

File: docs/pages/get-started/react-redux.mdx (hunk 3 above)
Comment (nvie) [reply docs/pages/get-started/react-redux.mdx:280]: @FlowFlorent I left this typo fix as well, think you missed it.

**Pull request merged**